.github/workflows/validate-cg-manifest.sh

```diff
@@ -13,6 +13,7 @@
 
 # Ignore some specs, mostly those with Source0 files that are not from an external source, or have very odd URLs
 ignore_list=" \
+  antlr \
```

Comment:
Are you sure we need to put these on the ignorelist?  This bypasses registration of these components with legal.  antlr, java-cup, and objectweb-asm don't seem to be ignorable items

Although I think there are a few others on this list too.

Reply:
The rationale was more along the lines of not knowing where the source tars come from. Cgmanifest needs a link to where we got the code from, but in cases like java-cup, this code doesn't seem to be hosted anywhere, or at least not in the right version (the tars' hashes don't match).

So a couple of questions about procedure here:
- What Source0 url, if any, should I use if the source tar is manually generated by some generate_tar.sh?
- What Source0 url, if any, should I use if there _is_ a source published online, but not in the version the spec is currently on?
- What Source0 url, if any, should I use if there is no external source to download from whatsoever?